lighthouse-core/config/budget.js

```diff
@@ -240,6 +240,38 @@ class Budget {
     };
   }
 
+  /**
+   * @param {string} origin
+   * @return {string}
+   */
+  static validateOrigin(origin) {
+    if (origin.endsWith(`/`)) {
+      throw new Error(`Invalid origin: ${origin}. Origin should not contain a trailing '/'.`);
```

Comment:
this feels like a very specific error message that would make more sense if I saw `firstPartyOrigins` being used :)